.github/workflows/prepare-for-minor-release.yml

```diff
@@ -0,0 +1,171 @@
+          event-type: az_quickstart_prep_for_minor_release
+          client-payload: '{"release_branch_name": "${{ inputs.release_branch_name }}" , "next_release_branch_alias_constraint": "${{needs.branch_name_prep.outputs.next_release_branch_alias_constraint}}", "main_branch_alias": "${{needs.branch_name_prep.outputs.main_branch_alias}}"}'
+
+  prepare_for_new_minor_release:
+    name: Prepare for a new minor release
+    runs-on: ubuntu-latest
+    needs: branch_name_prep
+    steps:
+    - name: Checkout az_quickstart
+      uses: actions/checkout@v4
+      with:
+        repository: az-digital/az_quickstart
+        ref: main
+
+    - name: Create and check out release branch
+      run: |
+        git checkout -b ${{ inputs.release_branch_name }}
+
+    - name: Check out new branch so we can make a pull request
+      run: |
+        git checkout -b ${{ inputs.release_branch_name }}-prepare-for-minor-release
+
+    - name: Update DDev config in release branch
+      run: |
+        NEW_VERSION=${{ inputs.release_branch_name }}
+        sed -i "0,/main/s//$NEW_VERSION/" .ddev/commands/web/install
+
+    - name: Update Security workflow yml in release branch
+      run: |
+        NEW_VERSION=${{ inputs.release_branch_name }}
+        sed -i "s/SCAFFOLD_BRANCH=main/SCAFFOLD_BRANCH=$NEW_VERSION/g" .github/workflows/security.yml
+
+    - name: Update Create Release workflow yml in release branch
+      run: |
+        NEW_VERSION=${{ inputs.release_branch_name }}
+        sed -i "s|git clone https://github.com/az-digital/az-quickstart-scaffolding.git az_quickstart|git clone -b $NEW_VERSION https://github.com/az-digital/az-quickstart-scaffolding.git az_quickstart|" .github/workflows/create-release.yml
+
+    - name: Update Lando config in release branch
+      run: |
+        NEW_VERSION=${{ inputs.release_branch_name }}
+        sed -i "s/SCAFFOLD_BRANCH=main/SCAFFOLD_BRANCH=$NEW_VERSION/g" .lando.yml
+
+    - name: Update Probo config in release branch
+      run: |
+        NEW_VERSION=${{ inputs.release_branch_name }}
+        sed -i "s/SCAFFOLD_BRANCH=main/SCAFFOLD_BRANCH=$NEW_VERSION/g" .probo.yaml
+
+    - uses: php-actions/composer@v6
+      with:
+        command: config extra.branch-alias.dev-main ${{ inputs.release_branch_name }}-dev
+
+    - name: Add changed files to release branch commit
+      run: |
+        git diff
+        git add .ddev/commands/web/install
+        git add .github/workflows/security.yml
+        git add .github/workflows/create-release.yml
+        git add .lando.yml
+        git add .probo.yaml
+        git add composer.json
+
+    - name: Set up git
+      run: |
+        git config user.name github-actions
+        git config user.email github-actions@github.com
+
+    - name: Push new release branch to GitHub
+      run: |
+        git commit --allow-empty -m "Prepare for new minor release ${{ inputs.release_branch_name }}"
+        git push origin -u ${{ inputs.release_branch_name }}
```

Comment:
This pushes the release branch back to the repo without any changes.

Reply:
This should be higher up, since it is adding to the commit in the Add changed files to release branch commit step above.